WireNetwork/Sources/WireNetwork/APIs/AuthenticationAPI/AuthenticationAPIV10.swift

```diff
@@ -21,4 +21,36 @@ import Foundation
 final class AuthenticationAPIV10: AuthenticationAPIV9 {
 
     override var apiVersion: APIVersion { .v10 }
+
+    override func getDomainRegistration(forEmail email: String) async throws -> DomainRegistrationConfiguration {
```

Comment:
@netbe or @dmitrysimkin do you know the feature where we will use this new endpoint?